### PR TITLE
Improve api-domain explanation

### PR DESCRIPTION
**Pull request by nottrobin**

### CODE REVIEW

File: upload-assets [hunk 1]
```diff
@@ -17,7 +17,7 @@ parser = argparse.ArgumentParser(
 )
 parser.add_argument(
     '-d', '--api-domain',
-    help='The domain for the API (https://<domain>}/v1/)',
+    help=f'The domain for the API. Default: ${api_domain}.',
```
Comment (anthonydillon) [inline upload-assets]: Do you need `$` I thought f-stings syntax was `help=f'The domain for the API. Default: {api_domain}.'`

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Question in the code.

File: upload-assets (hunk 1 above)
Comment (nottrobin) [reply upload-assets]: :facepalm:  Look at you teaching me Python.

Comment (nottrobin) [reply upload-assets]: Fixed

**Review by anthonydillon — APPROVED** [review]
Nicely done :+1:

**Pull request merged**